external/dynamo/.env.example

```
@@ -13,18 +13,32 @@
+
+# Required: Set your model directory path with model weights
+# EXAMPLE ls from properly configured directory
+# ~/models/Llama-3.3-70B-Instruct$ ls
+# LICENSE                           model-00003-of-00030.safetensors  model-00010-of-00030.safetensors  model-00017-of-00030.safetensors  model-00024-of-00030.safetensors  model.safetensors.index.json
+# README.md                         model-00004-of-00030.safetensors  model-00011-of-00030.safetensors  model-00018-of-00030.safetensors  model-00025-of-00030.safetensors  original
+# USE_POLICY.md                     model-00005-of-00030.safetensors  model-00012-of-00030.safetensors  model-00019-of-00030.safetensors  model-00026-of-00030.safetensors  special_tokens_map.json
+# config.json                       model-00006-of-00030.safetensors  model-00013-of-00030.safetensors  model-00020-of-00030.safetensors  model-00027-of-00030.safetensors  tokenizer.json
+# generation_config.json            model-00007-of-00030.safetensors  model-00014-of-00030.safetensors  model-00021-of-00030.safetensors  model-00028-of-00030.safetensors  tokenizer_config.json
+# model-00001-of-00030.safetensors  model-00008-of-00030.safetensors  model-00015-of-00030.safetensors  model-00022-of-00030.safetensors  model-00029-of-00030.safetensors
+# model-00002-of-00030.safetensors  model-00009-of-00030.safetensors  model-00016-of-00030.safetensors  model-00023-of-00030.safetensors  model-00030-of-00030.safetensors
 export DYNAMO_MODEL_DIR="/path/to/your/models/Llama-3.3-70B-Instruct"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)